common/services/move.test.js

```diff
@@ -1,8 +1,15 @@
+const proxyquire = require('proxyquire')
+
 const apiClient = require('../lib/api-client')()
 
-const moveService = require('./move')
 const personService = require('./person')
 
+const moveService = proxyquire('./move', {
+  'date-fns': {
+    formatISO: sinon.stub().returns('#timestamp'),
```

Comment:
We can probably not worry about mocking this method for the sake of not having to introduce proxyquire to this test.

Reply:
That seems...  back to front.

Reply:
https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/issues/495

Reply:
Added a test t ensure that `formatISO` is called with expected args